bears/xml2/XMLBear.py

```diff
@@ -50,17 +50,21 @@ class XMLBear:
     @staticmethod
     def create_arguments(filename, file, config_file,
                          xml_schema: path='',
-                         xml_dtd: path_or_url=''):
+                         xml_dtd: path_or_url='',
+                         xml_relaxng: path=''):
```

Comment:
Does it support a URL, like `xml_dtd`?  If so, this should be `path_or_url`, and the docstring updated.